cs/v1/CS_v1.0.0.yaml

```diff
@@ -120,28 +132,28 @@ paths:
           required: true
         - schema:
             type: string
-            format: date-time
-            example: '2021-04-01T14:12:56-01:00'
+            format: date
+            example: '2021-04-01'
+            pattern: '^[0-9]{4}-[0-9]{2}-[0-9]{2}$'
```

Comment:
These patterns are not needed as the `format: date` requires the input to be [ISO 8601 compliant](https://stackoverflow.com/questions/49379006/what-is-the-correct-way-to-declare-a-date-in-an-openapi-swagger-file). I don't think it hurts to have the pattern - it is though superfluous.
Check this out: https://swagger.io/docs/specification/data-models/data-types/#string